bundles/org.openhab.binding.huesync/README.md

```diff
@@ -0,0 +1,221 @@
+String    last_sync_mode_output            "Last Sync Mode"          <text>                                    {channel="huesync:box:LivingRoom:device-hdmi-out#mode"}
+
+//Commands
+String    huesync_mode                     "Mode"                    <switch>                                  {channel="huesync:box:LivingRoom:device-commands#mode"}
+Switch    sync_active                      "Sync active"             <switch>                                  {channel="huesync:box:LivingRoom:device-commands#sync-active"}
+Switch    hdmi_active                      "HDMI active"             <switch>                                  {channel="huesync:box:LivingRoom:device-commands#hdmi-active"}
+String    hdmi_source                      "HDMI Source"             <player>                                  {channel="huesync:box:LivingRoom:device-commands#hdmi-source"}
+Dimmer    huesync_brightness               "Brightness"              <slider>                                  {channel="huesync:box:LivingRoom:device-commands#brightness"}
+
+```
+
+#### example.sitemap
+
+```java
+sitemap demo label="Hue Sync Box" {
+ Frame {
+  Group item=firmware
+ }
+ Frame label="Commands" icon=settings {
+  Text item=huesync_mode                     
+  Text item=hdmi_active                    
+  Switch item=sync_active  
+  Text item=hdmi_source  
+  Buttongrid label="HDMI Source" staticIcon=player {
+   Button row=1 column=1 item=hdmi_source label="Source 1" stateless click=input1
+   Button row=2 column=1 item=hdmi_source label="Source 2" stateless click=input2
+   Button row=3 column=1 item=hdmi_source label="Source 3" stateless click=input3
+   Button row=4 column=1 item=hdmi_source label="Source 4" stateless click=input4
+  }
+  Selection item=hdmi_source mappings=[input1="Source 1", input2="Source 2", input3="Source 3", input3="Source 4"]                   
+  
+  Slider item=huesync_brightness minValue=0 maxValue=200 step=10
+ }
+ Frame label="HDMI Inputs 1 & 2" icon="iconify:mdi:hdmi-port" {
+  Default item=hdmi_in1
+
+  Group item=hdmi_in2 label="HDMI 2" icon="iconify:mdi:hdmi-port" {
+   Default item=friendly_name_input2 icon="iconify:mdi:text"
+   Default item=friendly_type_input2 icon="iconify:mdi:devices"
+   Default item=hdmi_connection_status_input2 icon="iconify:mdi:connection"
+   Default item=last_sync_mode_input2 icon="iconify:mdi:multimedia"
+  }
+ }        
+ Frame label="HDMI 3" icon=player {
+  Default item=friendly_name_input3            
+  Default item=friendly_type_input3            
+  Default item=hdmi_connection_status_input3   
+  Default item=last_sync_mode_input3 
+ } 
+ Frame label="HDMI 4" icon=player {
+  Text item=friendly_name_input4            
+  Text item=friendly_type_input4            
+  Text item=hdmi_connection_status_input4   
+  Text item=last_sync_mode_input4
+ }
+`
```

Comment:
```suggestion
```
```

Reply:
Not sure if this suggestion works, but i meant to fix the code close tags with three times a `